include/erlcloud_ddb_streams.hrl

```diff
@@ -36,8 +36,24 @@
 -record(ddb_streams_describe_stream,
         {stream_description :: #ddb_streams_stream_description{}
         }).
+-record(ddb_streams_record_dynamodb,
```

Comment:
The names `ddb_streams_stream`, `ddb_streams_stream_description`, `ddb_streams_stream_record` and `ddb_streams_record` are a little complex to understand, indeed! I'll think it over.